_layouts/docs.html

```diff
@@ -40,5 +40,5 @@
 
 base: /docs/
 ---
-
+{% include edit.html %}
```

Comment:
I'd prefer a shorter version at the top that does not have the "and submit a pull request!"